astroquery/gaia/tests/test_gaiatap.py

```diff
@@ -16,6 +16,8 @@
 """
 import unittest
 import os
+import pytest
```

Comment:
We still import it from astropy, but I have an open PR that will change that to import directly again. Thanks for noting this though.